.github/workflows/ci.yml

```
@@ -13,80 +13,54 @@ jobs:
     name: run tests
     strategy:
       matrix:
-        platform: [ubuntu-latest, macos-latest, windows-latest]
-        rust: ['1.62', stable]
+        platform: [ubuntu-latest, macos-latest, windows-latest, ubuntu-20.04]
+        toolchain: [stable, 1.62.1]
     runs-on: ${{ matrix.platform }}
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@stable
```

I think the intended way is to use `@master` when specifying `toolchain:` below

---

```suggestion
      - uses: dtolnay/rust-toolchain@master
```
Will update it to `@master` above. Regarding the toolchain below, the detail is specified in the top as a matrix job instead of specifying it multiple times in the file.